.gitignore

```diff
@@ -0,0 +1,8 @@
+*.o
+*.lo
+*.la
+*.lai
+*.so
+Makefile
+Makefile.in
+*.m4
```

Comment:
Having myself removed .gitignore from here as a nuisance, I'm not too fond of getting such things back in.

Reply:
Dropped.